### PR TITLE
Add CLI examples for CloudWatch Synthetics

### PR DESCRIPTION
**Pull request by ashishbharadwajm**
*Issue #, if available:*

*Description of changes:*

Adding example CLI commands for Amazon CloudWatch Synthetics: https://docs.aws.amazon.com/cli/latest/reference/synthetics/

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: awscli/examples/synthetics/start-canary.rst [hunk 1]
```diff
@@ -0,0 +1,10 @@
+**To Run a canary **
+
+The following ``start-canary`` example runs a canary named ``demo_canary``.
```
Comment (elysahall) [inline awscli/examples/synthetics/start-canary.rst:3]:
```suggestion
The following ``start-canary`` example runs a canary named ``demo_canary``. ::
```

File: awscli/examples/synthetics/stop-canary.rst [hunk 2]
```diff
@@ -0,0 +1,10 @@
+**To Stop a canary **
```
Comment (elysahall) [inline awscli/examples/synthetics/stop-canary.rst:1]:
```suggestion
**To stop a canary **
```

File: awscli/examples/synthetics/stop-canary.rst [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+
+The following ``stop-canary`` example stops the canary named ``demo_canary``.
```
Comment (elysahall) [inline awscli/examples/synthetics/stop-canary.rst:3]:
```suggestion
The following ``stop-canary`` example stops the canary named ``demo_canary``. ::
```

File: awscli/examples/synthetics/tag-resource.rst [hunk 4]
```diff
@@ -0,0 +1,20 @@
+**Example 1: To Assign a tag to the canary**
```
Comment (elysahall) [inline awscli/examples/synthetics/tag-resource.rst:1]:
```suggestion
**Example 1: To assign a tag to the canary**
```

File: awscli/examples/synthetics/tag-resource.rst [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+
+The following ``tag-resource`` example assigns a tag to the canary named ``demo_canary``.
```
Comment (elysahall) [inline awscli/examples/synthetics/tag-resource.rst:3]:
```suggestion
The following ``tag-resource`` example assigns a tag to the canary named ``demo_canary``. ::
```

File: awscli/examples/synthetics/tag-resource.rst [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+    aws synthetics tag-resource \
+        --resource-arn arn:aws:synthetics:us-east-1:123456789012:canary:demo_canary \
+        --tags blueprint=heartbeat
+
+This command produces no output.
+
+**Example 2: To Assign a tag to the group**
```
Comment (elysahall) [inline awscli/examples/synthetics/tag-resource.rst:10]:
```suggestion
**Example 2: To assign a tag to the group**
```

File: awscli/examples/synthetics/tag-resource.rst [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+
+The following ``tag-resource`` example assigns a tag to the group named ``demo_group``.
```
Comment (elysahall) [inline awscli/examples/synthetics/tag-resource.rst:12]:
```suggestion
The following ``tag-resource`` example assigns a tag to the group named ``demo_group``. ::
```

File: awscli/examples/synthetics/untag-resource.rst [hunk 8]
```diff
@@ -0,0 +1,21 @@
+**Example 1: To Remove a tag from the canary**
```
Comment (elysahall) [inline awscli/examples/synthetics/untag-resource.rst:1]:
```suggestion
**Example 1: To remove a tag from the canary**
```

File: awscli/examples/synthetics/untag-resource.rst [hunk 9]
```diff
@@ -0,0 +1,21 @@
...
+
+The following ``untag-resource`` example removes a tag from the canary named ``demo_canary``.
```
Comment (elysahall) [inline awscli/examples/synthetics/untag-resource.rst:3]:
```suggestion
The following ``untag-resource`` example removes a tag from the canary named ``demo_canary``. ::
```

File: awscli/examples/synthetics/untag-resource.rst [hunk 10]
```diff
@@ -0,0 +1,21 @@
...
+
+    aws synthetics untag-resource \
+        --resource-arn arn:aws:synthetics:us-east-1:123456789012:canary:demo_canary \
+        --tag-keys blueprint
+
+This command produces no output.
+
+**Example 2: To Remove a tag from the group**
```
Comment (elysahall) [inline awscli/examples/synthetics/untag-resource.rst:11]:
```suggestion
**Example 2: To remove a tag from the group**
```

File: awscli/examples/synthetics/untag-resource.rst [hunk 11]
```diff
@@ -0,0 +1,21 @@
...
+
+The following ``untag-resource`` example assigns a removes a tag from the group named ``demo_group``.
```
Comment (elysahall) [inline awscli/examples/synthetics/untag-resource.rst:13]:
```suggestion
The following ``untag-resource`` example assigns a removes a tag from the group named ``demo_group``. ::
```

**Review by elysahall — CHANGES_REQUESTED** [review]
Hi Ashish,
Thank you for your command example submission. I've finished reviewing your pull request. Please fix these formatting issues and push your updated commit. Thanks!

**Comment by ashishdhingra** [comment]
@elysahall I guess you tagged wrong person. Please reach out to @ashishbharadwajm.

**Comment by ashishbharadwajm** [comment]
> Hi Ashish, Thank you for your command example submission. I've finished reviewing your pull request. Please fix these formatting issues and push your updated commit. Thanks!

Thank you for the review. Published the new commit with the suggested changes.

**Pull request closed without merging**